src/OpenTelemetry.Exporter.Prometheus/CHANGELOG.md

```diff
@@ -5,6 +5,8 @@
 * Added `IApplicationBuilder` extension methods to help with Prometheus
   middleware configuration on ASP.NET Core
   ([#3029](https://github.com/open-telemetry/opentelemetry-dotnet/pull/3029))
+* Changed Prometheus exporter to return 204 No Content and log a warning event
```

Comment:
nit: I think we typically do a blank line in between each log entry.

Reply:
Added blank.